WordPress/Classes/ViewRelated/Activity/ActivityListViewController.swift

```diff
@@ -266,6 +266,8 @@ private extension ActivityListViewController {
 
         if noResultsViewController.view.superview != tableView {
             tableView.addSubview(withFadeAnimation: noResultsViewController.view)
+        } else if noResultsViewController.titleLabel.text != viewModel.titleText {
+            noResultsViewController.view.fadeInWithAnimation()
```

Comment:
The fade sort of hides the shifting, but it looks a bit odd to my eye.  Maybe because the graphic is the same?  I wonder if rather than a fade animation if it would look better to set the subtitle to an empty string?  That would prevent the vertical shift of the form (assuming the title is only just one line) and there'd be less happening visually. 
Thoughts?